.changeset/rotten-maps-change.md

```diff
@@ -0,0 +1,6 @@
+---
+'tinacms': patch
+'@tinacms/cli': patch
+---
+
+Client caching
```

Comment:
```suggestion
CLI - Adds client caching and cli flag to disable: --no-client-build-cache
```

These will need to be more details as changelogs will be cross published to https://tina.io/whats-new/tinacms/

I'd recommend tagging the PR and checking build times against https://github.com/sswconsulting/ssw.website - if there are any performance improvements, include stats as well